packages/common/src/additions.schema.graphql

```diff
@@ -314,7 +314,7 @@ union RequisitionsResponse = ConnectorError | RequisitionConnector
 
 type RequisitionConnector {
   totalCount: Int!
-  nodes: [RequisitionNode]!
+  nodes: [RequisitionNode!]!
```

Comment:
huh. so the `RequisitionNode` is required and the `nodes` prop is also required? is that what this means?

Reply:
Yeah.. that's how the backend does it in the schema also, I did it incorrectly for this. The type being generated was an array of `RequisitionNode | null` (and removing the final ! meant nodes: Array<RequisitionNode | null> | null).. so to get Array<RequisitionNode> needs `!` in both places